src/pybind/mgr/dashboard/module.py

```diff
@@ -821,6 +831,14 @@ def get_perf_schema(self, **args):
                         ret[k1][k2] = sorted_dict
                 return ret
 
+        url_prefix = self.get_localized_config('url_prefix', '')
```

Comment:
I guess this isn't something one would configure differently on different mgr daemons in the same cluster, so use a get_config rather than get_localized_config